docs/CHANGES.rst

```diff
@@ -0,0 +1,3 @@
+#######
```

Comment:
You want this to be an empty files because towncrier is going to add things to the top each time it runs.